pkg/corroborate/renderer/index.html

```
@@ -791,8 +1048,10 @@
     </div>
   </div>
   <div class="nav-toggles">
-    <button class="btn" id="toggle-nav" title="show/hide the dashboards panel">☰ dashboards</button>
-    <button class="btn" id="toggle-filters" title="show/hide recipe selectors &amp; evidence filters">⚙ filters</button>
+    <button class="btn" id="toggle-nav" title="show or hide the navigation sidebar"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)